sources/assets/Xenko.Core.Assets.CompilerApp/app.config

```diff
@@ -22,10 +22,6 @@
         <assemblyIdentity name="Microsoft.Build.Tasks.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-15.1.0.0" newVersion="15.1.0.0" />
       </dependentAssembly>
-      <dependentAssembly>
-        <assemblyIdentity name="System.Runtime.InteropServices.RuntimeInformation" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.0.2.0" newVersion="4.0.2.0" />
-      </dependentAssembly>
```

Comment:
If I revert commit 1318967 then `git clean -fxd` and build the solution I get the following for each affected project:
```
83>    C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3277: Found conflicts between different versions of "System.Runtime.InteropServices.RuntimeInformation" that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed.
```
Removing the binding redirects for `System.Runtime.InteropServices.RuntimeInformation` was the first thing I tried that fixed the warnings, so it could be misleading.

I used the same approach in commit fece0ea, although in that case there were more warnings that the binding redirects specified in `app.config` conflicted with the ones being generated automatically. I changed the versions in `app.config` to match but later I removed them altogether since the other warnings highlighted that there was no need to specify them in the first place (as they were being generated automatically).

[DetailedBuild.zip](https://github.com/xenko3d/xenko/files/2282460/DetailedBuild.zip)

Reply:
Also, the following projects are explicitly [enabling automatic binding redirects](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/how-to-enable-and-disable-automatic-binding-redirection#enabling-automatic-binding-redirects-manually):
```
sources/tools/Xenko.VisualStudio.PackageInstall/Xenko.VisualStudio.PackageInstall.csproj:6:    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
sources/tools/Xenko.PackageInstall/Xenko.PackageInstall.csproj:6:    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
sources/engine/Xenko.Assets/Xenko.Assets.csproj:9:    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
sources/editor/Xenko.GameStudio/Xenko.GameStudio.csproj:12:    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
sources/core/Xenko.Core.AssemblyProcessor.Tests/Xenko.Core.AssemblyProcessor.Tests.csproj:6:    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
sources/core/Xenko.Core.AssemblyProcessor/Xenko.Core.AssemblyProcessor.csproj:7:    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```
Although I don't think that's necessary when targeting .NET FW 4.5.1+; not completely sure though.

Reply:
OK I will test with that change and if it works let's keep it that way.

Reply:
FYI I had to revert the app.config files, some people had problem building Xenko.
Let's fix that later as part of #12 (build in separate folders rather than common folder)